lib/create-testing-library-rule/detect-testing-library-utils.ts

```diff
@@ -214,6 +212,7 @@ export function detectTestingLibraryUtils<
 
 			const originalNodeName =
 				isImportSpecifier(importedUtilSpecifier) &&
+				ASTUtils.isIdentifier(importedUtilSpecifier.imported) &&
```

Comment:
Extra check is needed because of changes from https://github.com/typescript-eslint/typescript-eslint/pull/9972: `imported` can now be `Identifier | StringLiteral`, which was only `Identifier` before